blackbox/docs/admin/system-information.rst

```diff
@@ -1091,6 +1091,30 @@ statements.
 | ``percentiles``              | An object containing different percentiles         | ``OBJECT``       |
 +------------------------------+----------------------------------------------------+------------------+
 
+Classification
+..............
+
+Certain statement types (such as ``SELECT`` statements) have additional labels
+in their classification. These labels are the names of the logical plan
+operators that are involved in the query.
```

Comment:
can we have another sentence on what logical plan operaters exist, can we link to some code on github or further documentation?

Reply:
These operators are internals and always subject to change even there are no user-facing changes. These parts should not be documented in the user documentation.
I think people may get an idea what these operators do/are when they see their names in the labels column. Most of them are self-explanatory.